armi/tests/tutorials/c5g7-blueprints.yaml

```diff
@@ -162,7 +162,6 @@ blocks:
             # has same od as fuel, so there needs to be something in the gap.
             shape: Circle
             material: Void
-            isotopics: moderator
```

Comment:
Sorry, honest question, why did we need to make this change?

(Just trying to wrap my head around the PR.  I am not worried either way.)

Reply:
This PR does not allow density to be assigned to Void and throws an error if a user attempts to do so. Previously, this would weakly fail, emitting a warning that nothing was done.

The reason for throwing an error (aside from it being silly to assign properties to Void) was to avoid dividing by zero.